spec/routing/female_routing_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'prisoner routes', type: :routing do
+  describe get: '/prisons/PFI/prisoners/allocated' do
+    context 'with switch on' do
+      let(:test_strategy) { Flipflop::FeatureSet.current.test! }
+
+      before do
+        test_strategy.switch!(:womens_estate, true)
+      end
+
+      after do
+        test_strategy.switch!(:womens_estate, false)
+      end
+
+      it { is_expected.to route_to controller: 'female_prisoners', action: 'allocated', prison_id: 'PFI' }
+    end
+
+    context 'without switch' do
+      it { is_expected.to route_to controller: 'summary', action: 'allocated', prison_id: 'PFI' }
+    end
+  end
+
+  describe get: '/prisons/LEI/prisoners/allocated' do
```

Comment:
Might be nice to add comments or context blocks to make it really clear that `LEI` is a **male closed** prison, and `PFI` is a **female** prison.